server/auth/types/basic/routes.ts

```diff
@@ -186,7 +186,9 @@ export class BasicAuthRoutes {
           }
           context.security_plugin.logger.info('The Redirect Path is ' + redirectUrl);
           try {
-            user = await this.securityClient.authenticateWithHeaders(request, {});
+            user = await this.securityClient.authenticateWithHeaders(request, {
+              _auth_request_type_: 'anonymous',
```

Comment:
this allows specifying login requests for anonymous login to be of anonymous type. However, the question I'm trying to resolve is, is this feasible approach? Would every request coming from anonymous user when intercepted by backend require this header to be present?

Reply:
solved by adding dummy credentials for anonymous user.